app/models/facility.rb

```diff
@@ -2,6 +2,10 @@
 
 class Facility < ApplicationRecord
 
+  # Migration is required if Paperclip is being used.
+  # The image is used in the UI if the facility_tile_list feature flag is enabled
+  include DownloadableFiles::Image
```

Comment:
I think this needs a feature flag, right?  Maybe the code comment would be more at home in the Image concern?